pkg/repository/aws/crowdactionParticipationManager.go

```diff
@@ -0,0 +1,33 @@
+package aws
```

Comment:
Instead of creating a new file for the crowdactionParticipationManager, I would try to put this method in the dynamo.go file. Maybe I misunderstood the concept of the repositories, but I thought they were supposed to be revolving around the AWS services as opposed to the business interactions. Ultimately it'd an interesting discussion to have.

Reply:
I was thinking about it more in terms of DDD (https://medium.com/@pererikbergman/repository-design-pattern-e28c0f3e4a30). @mrsoftware what was your initial idea?  
Maybe we could use generic repositories for the database, but we should decide it once and then apply it consistently.

Reply:
I think this relates to the recent comments by @mrsoftware on #119.

Reply:
All righty then!